inference-engine/tests/functional/plugin/myriad/subgraph_tests/dsr_binary_elementwise.cpp

```diff
@@ -93,20 +114,29 @@ INSTANTIATE_TEST_CASE_P(DISABLED_DynamicBinaryElementwise, DSR_BinaryElementwise
         ::testing::Values(CommonTestUtils::DEVICE_MYRIAD)));
 
 TEST_P(DSR_BinaryElementwiseSingleDSR, CompareWithReference) {
+    if (doNotSupportI32.count(m_eltwiseType) && m_inDataType == ngraph::element::i32) {
+        SKIP() << "Eltwise Power doesn't support int32_t inputs" << std::endl;
```

Comment:
Why you're mentioning `Power` in error message?

Reply:
Fixed